lib/ansible/plugins/inventory/kubevirt.py

```diff
@@ -0,0 +1,219 @@
+# Copyright (c) 2018 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = '''
+    name: kubevirt
+    plugin_type: inventory
+    author:
+      - KubeVirt Team (@kubevirt)
+
+    short_description: KubeVirt inventory source
+
+    description:
+      - Fetch running VirtualMachines for one or more namespaces.
+      - Groups by namespace, namespace_vms  and labels.
+      - Uses kubevirt.(yml|yaml) YAML configuration file to set parameter values.
+
+    options:
+      plugin:
+        description: token that ensures this is a source file for the 'kubevirt' plugin.
+        required: True
+        choices: ['kubevirt']
+        type: str
+      connections:
+          type: list
+          description:
+          - Optional list of cluster connection settings. If no connections are provided, the default
+            I(~/.kube/config) and active context will be used, and objects will be returned for all namespaces
+            the active user is authorized to access.
+          name:
```

Comment:
you might want a `suboptions` key here,  they are not implemented in config (yet) but eventually they should work the same as module suboptions.

Reply:
done